content/influxdb/v2.1/write-data/oss-to-cloud.md

```diff
@@ -0,0 +1,260 @@
+example-m,exampleTag=A temp=80.3,rpm=4542i 1609459260000000000
+example-m,exampleTag=A temp=81.1,rpm=4901i 1609459320000000000
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+#### Use experimental.to()
+Use `experimental.to()` to write data with column names that should be used as
+field keys and column values that should be used as field values.
+
+- `experimental.to()` requires `_time` and `_measurement` columns.
+- Columns **in** the [group key](/flux/v0.x/get-started/data-model/#grouop-key)
+  (other than `_measurement`) are parsed as tags where the column name is the
+  tag key and the column value is the tag value.
+- Columns **not in** the group key are parsed as fields where the column name is 
+  the field key and the column value is the field value.
+
+{{< expand-wrapper >}}
+{{% expand "View example `experimental.to()` data" %}}
+**Input data**
+{{< flux/group-key "[_measurement, exampleTag]">}}
+| _time                | _measurement | exampleTag | temp |  rpm |
+| :------------------- | :----------- | :--------: | ---: | ---: |
+| 2021-01-01T00:00:00Z | example-m    |     A      | 80.0 | 4023 |
+| 2021-01-01T00:01:00Z | example-m    |     A      | 80.3 | 4542 |
+| 2021-01-01T00:02:00Z | example-m    |     A      | 81.1 | 4901 |
+
+**Output line protocol**
+```
+example-m,exampleTag=A temp=80.0,rpm=4023i 1609459200000000000
+example-m,exampleTag=A temp=80.3,rpm=4542i 1609459260000000000
+example-m,exampleTag=A temp=81.1,rpm=4901i 1609459320000000000
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+## Selectively write data to InfluxDB Cloud
+`to()` and `experimental.to()` write data as it is pipe-forwarded.
+To selective write data to InfluxDB, use `filter()` to filter points and 
+pipe-forward the filtered data into `to()`.
+
+```js
+import "influxdata/influxdb/secrets"
+
+cloudToken = secrets.get(key: "INFLUX_CLOUD_API_TOKEN")
+
+from(bucket: "example-oss-bucket")
+    |> range(start: -10m)
+    |> filter(fn: (r) => r._measurement == "example-measurement")
+    |> filter(fn: (r) => r.exampleTag == "someTagValue")
+    |> filter(fn: (r) => r._field == "example-field")
+    |> filter(fn: (r) => r._value >= 80.0)
+    |> to(
+        bucket: "example-cloud-bucket",
+        host: "https://cloud2.influxdata.com",
+        org: "example-org",
+        token: cloudToken,
+    )
+```
+
+## Process data and write it to InfluxDB Cloud
+`to()` and `experimental.to()` write data as it is pipe-forwarded.
+Use [`map()`](/flux/v0.x/stdlib/universe/map/), [`set()`](/flux/v0.x/stdlib/universe/set/),
+[`aggregateWindow()`](/flux/v0.x/stdlib/universe/aggregatewindow/)
+[aggregate](/flux/v0.x/function-types/#aggregates) or
+[selector functions](/flux/v0.x/function-types/#selectors), or other Flux
+functions to process and shape data as needed.
+
+{{% note %}}
+The shape of the processed data determines which `to()` function to use.
+_See [Identify which to() function to use](#identify-which-to-function-to-use)_. 
+{{% /note %}}
```

Comment:
feels a bit circular/overlapping 

Ideally, we'd link to a separate procedure for info about "processing" data that covers all aforementioned functions. 

Perhaps eventually here :
(/influxdb/cloud/process-data/get-started/#process-or-transform-your-data)...once we land examples for map/set/aggregrate/selectors?